### PR TITLE
UI Fix

### PR DESCRIPTION
**Pull request by sp4ce-cowboy**
- Remove summary tab
- CommandBox fix (and UI test)
- Fix issue with large amounts in Rolling Balance indicator (and UI Test)

### CODE REVIEW

File: src/main/java/unicash/ui/StatusBarFooter.java [hunk 1]
```diff
@@ -94,9 +93,11 @@ public void updateBalance(ObservableList<Transaction> transactions) {
         }
 
         if (Double.compare(balance, 0) < 0) {
-            balanceString = NEGATIVE_BALANCE_STRING + new Amount(Math.abs(balance));
+            double formattedNegativeBalance = Math.round((Math.abs(balance) * 100.0 / 100.0));
+            balanceString = NEGATIVE_BALANCE_STRING + String.format("%.2f", formattedNegativeBalance);
         } else {
-            balanceString = POSITIVE_BALANCE_STRING + new Amount(balance);
+            double formattedPositiveBalance = Math.round(balance * 100.0 / 100.0);
+            balanceString = POSITIVE_BALANCE_STRING + String.format("%.2f", formattedPositiveBalance);
```
Comment (woojiahao) [inline src/main/java/unicash/ui/StatusBarFooter.java:100]: Since `%.2f` does rounding, do you still need the manual rounding?

**Review by woojiahao — CHANGES_REQUESTED** [review]
Minor comment on using `%.2f`

File: src/main/java/unicash/ui/StatusBarFooter.java (hunk 1 above)
Comment (sp4ce-cowboy) [reply src/main/java/unicash/ui/StatusBarFooter.java:100]: Oh that's right, let me get that fixed

File: src/main/java/unicash/ui/StatusBarFooter.java [hunk 2]
```diff
@@ -94,9 +93,11 @@ public void updateBalance(ObservableList<Transaction> transactions) {
         }
 
         if (Double.compare(balance, 0) < 0) {
-            balanceString = NEGATIVE_BALANCE_STRING + new Amount(Math.abs(balance));
+            double formattedNegativeBalance = (Math.abs(balance) * 100.0 / 100.0);
```
Comment (woojiahao) [inline src/main/java/unicash/ui/StatusBarFooter.java:96]: Hm, wouldn't `* 100.0 / 100.0` just be `* 1` haha, can probably just use `Math.abs(balance)`

**Review by woojiahao — CHANGES_REQUESTED** [review]
Minor issue with `* 100 / 100`

**Review by woojiahao — APPROVED** [review]
LGTM, great work resolving these issues

**Pull request merged**